src/test/java/persistence/PersistenceContextImplTest.java

```diff
@@ -0,0 +1,52 @@
+package persistence;
+
+import entity.Person;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.stream.IntStream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/*
+- 영속성 컨텍스트에 Entity객체를 저장 후 저장되어있는 Entity 객체를 가져온다.
+- 영속성 컨텍스트에 저장되어있는 Entity 객체를 제거한다.
+*/
+class PersistenceContextImplTest {
+
+    @DisplayName("영속성 컨텍스트에 Entity객체를 저장 후 저장되어있는 Entity 객체를 가져온다..")
+    @Test
+    void insertFindTest() {
+        PersistenceContextImpl persistenceContext = new PersistenceContextImpl();
+
+        Person person = createPerson(1);
+
+        EntityInfo<Person> entityInfo = new EntityInfo<>(1, Person.class);
+
+        persistenceContext.insertEntity(entityInfo, person);
+
+        assertThat(persistenceContext.findEntity(new EntityInfo<>(1, Person.class)))
+                .extracting("id", "name", "age", "email")
+                .contains(1L, "test1", 29, "test@test.com");
```

Comment:
```suggestion
        Person person = createPerson(1);
        EntityInfo<Person> entityInfo = new EntityInfo<>(1, Person.class);
        persistenceContext.insertEntity(entityInfo, person);

        var actual = persistenceContext.findEntity(new EntityInfo<>(1, Person.class));
        assertThat(actual).extracting("id", "name", "age", "email")
                .contains(1L, "test1", 29, "test@test.com");
```
1. 각 칸을 띄워두면 메소드 길이만 늘어나서 오히려 가독성이 좋지 않아 적절하게 공백을 활용하시는 것이 좋습니다 :) 
2. `PersistenceContextImpl`를 모든 테스트에서 사용할 것이라면 선언자체는 클래스단에 해주시는 게 더 좋을 것 같네요.
3. "어떤 동작을 테스트하고자하는가(when)"와 "어떻게 동작하길 바라는가(then)"는 구분되면 좀 더 가독성이 높을 것 같네요. 변수선언 없이 코드가 짧은 것보다는 좀더 명확한 것이 좋습니다.